src/de/jost_net/JVerein/gui/view/EinstellungenAbrechnungView.java

```diff
@@ -45,6 +45,8 @@ public void bind() throws Exception
         control.getCt1SepaVersion());
     cont.addLabelPair("SEPA XML-Version - Lastschrift",
         control.getSepaVersion());
+    cont.addLabelPair("Verechnungs-Konto f�r Lastschriften",
+        control.getVerechnungskonto());
```

Comment:
Verrechnungskonto bitte mit Doppel-r schreiben, sowohl im Text, als auch in der Funktion.